docs/server/settings.rst

```diff
@@ -605,3 +579,11 @@ Deprecated Settings
      To simplify checker code, the ability to have a custom quality checker was
      removed.  To create custom checks, write them within the Translate
      Toolkit.
+
+
+.. setting:: POOTLE_SCORE_COEFFICIENTS
+
+``POOTLE_SCORE_COEFFICIENTS``
+  .. deprecated:: 2.8
```

Comment:
It is not deprecated but removed. This might be misleading in the way that people might think is not yet gone.